### PR TITLE
Added Documentation for Frappe Accounting

### PR DESCRIPTION
**Pull request by sahil28297**

### CODE REVIEW

**Comment by rmehta** [comment]
screenshots?

File: frappe_io/www/accounting/docs/chartOfAccounts.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<!-- add-next-prev-links -->
+## Chart of Accounts
+
+After completing the setup, you are presented with Chart of Accounts which is bootstrapped for you based on the country you selected in the Setup Wizard.
+
+The Chart of Accounts forms the blueprint of your organization. The overall structure of your Chart of Accounts is based on a system of double entry accounting that has become a standard all over the world to quantify how a company is doing financially.
+
+The Chart Of Accounts is displayed in a Tree format according to the Parent-Child relationship among the accounts. You can add/remove any Accounts as you like.
+
+The name of your company is seen at the top of the tree. You can view the children of the root by clicking on it. The children of a particular root can be seen at a greater indentation level than the root.
```
Comment (netchampfaris) [inline frappe_io/www/accounting/docs/chartOfAccounts.md]: can remove this paragraph

File: frappe_io/www/accounting/docs/generalLedger.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<!-- add-next-prev-links -->
+## General Ledger
+
+A general ledger represents the formal ledger for a company's financial statements with debit and credit account records validated by a trial balance. The ledger provides a complete record of financial transactions over the life of the company. The ledger holds account information that is needed to prepare financial statements and includes accounts for assets, liabilities, owners' equity, revenues and expenses. In simple terms, General Ledger is a detailed report for all transactions posted to each account and for every transaction there is a Credit and Debit account so it lists them all up.
+
+##### Viewing General Ledger :
```
Comment (netchampfaris) [inline frappe_io/www/accounting/docs/generalLedger.md]: no space before `:`

File: frappe_io/www/accounting/docs/index.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
+<!-- add-next-prev-links -->
+
+# Frappe Accounting
+Simple desktop accounting software for everyone.
+
+---
+
+## What is Frappe Accounting?
+Frappe Accounting is a simple JavaScript based software designed for business owners to manage their accounting and finances. Frappe Accounting is based on FrappeJS which is inspired by the Frappe Framework. Frappe Accounting works offline, and stores the data in a local file which is portable. It works on all the major operating systems like macOS, Linux and Windows.
```
Comment (netchampfaris) [inline frappe_io/www/accounting/docs/index.md]: add link to FrappeJS homepage on FrappeJS word

File: frappe_io/www/accounting/docs/index.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+## Features
+
+- Chart Of Accounts
+- Item Management
+- Party Management
+- Taxation
+- Invoicing
+- Payment Handler
+- General ledger Report
```
Comment (netchampfaris) [inline frappe_io/www/accounting/docs/index.md]: Financial Reporting instead of specific reports

File: frappe_io/www/accounting/docs/index.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+Download the executable based on your operating systems below.
+
+##### Frappe Accounting Beta 0.0.2
+- Linux - [frappe-accounting-0.0.2-x86_64.AppImage](https://github.com/frappe/accounting/releases/download/0.0.2/frappe-accounting-0.0.2-x86_64.AppImage)
+- MacOS - [Frappe.Accounting-0.0.2.dmg](https://github.com/frappe/accounting/releases/download/0.0.2/Frappe.Accounting-0.0.2.dmg)
+- Windows - [Frappe.Accounting.Setup.0.0.2.exe](https://github.com/frappe/accounting/releases/download/0.0.2/Frappe.Accounting.Setup.0.0.2.-.fixed.exe)
+
+---
+
+## Licence
```
Comment (netchampfaris) [inline frappe_io/www/accounting/docs/index.md]: License

File: frappe_io/www/accounting/docs/payment.md [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+
+##### Payment Modal :
+- By clicking on the payment option, a payment modal will appear in which most of the details will already be fetched from the invoice.
+
+<img  src="/accounting/assets/img/paymentmodal.png"
+      class="screenshot"
+/>
+
+##### Payment Submission :
+- By clicking on the save button, payment will be completed.
```
Comment (netchampfaris) [inline frappe_io/www/accounting/docs/payment.md]: Add this: "You can check the corresponding ledger entries by clicking on **Ledger Entries** in the actions dropdown"

**Pull request merged**